iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/DanglingLineAdderImpl.java

```diff
@@ -64,10 +64,6 @@ public GenerationAdder setTargetV(double targetV) {
 
         @Override
         public DanglingLineAdder add() {
-            ValidationUtil.checkActivePowerLimits(DanglingLineAdderImpl.this, minP, maxP);
-            ValidationUtil.checkActivePowerSetpoint(DanglingLineAdderImpl.this, targetP);
-            ValidationUtil.checkVoltageControl(DanglingLineAdderImpl.this, voltageRegulationOn, targetV, targetQ);
```

Comment:
This is arguable: I think we should keep the tests here: It's not possible to update the GenerationAdder after you have called the add(). So you should be noticed that it's not complete when we consider it should be fully completed.
@MioRtia @tadam50 What do you think?

Reply:
My mistake. I'm agree with you. I think validation checks must be done as soon as possible.